c#/Mandoline.Api.Examples/Client.Repl/Program.cs

```diff
@@ -8,20 +8,50 @@ namespace Client.Repl
 {
     using System;
     using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
     using System.Linq;
     using System.Text;
     using System.Threading.Tasks;
+    using System.Xml.Linq;
     using Core;
-    using Replify;
-
+    using Sharprompt;
     internal class Program
     {
-        private static void Main(string[] args)
+
+        private static async Task Main(string[] args)
         {
             // Entry point fo REPL command line interface
-            var repl = new ClearScriptRepl();
-            repl.AddHostType("Console", typeof(Console));
-            repl.StartReplLoop();
+
+            var commands = new Dictionary<string, Func<Task>>();
+
+            var userCommands = new UserCommand();
+            commands.Add("GetUser", userCommands.GetUser);
```

Comment:
might be a bit better to use `nameof(userCommands.GetUser)` instead of pure strings, just a suggestion